lib/kb_uploadmethods/kb_uploadmethodsImpl.py

```diff
@@ -126,7 +126,7 @@ def upload_fastq_file(self, ctx, params):
         # return the results
         return [returnVal]
 
-    def upload_fasta_gff_file(self, ctx, params):
+    def import_fasta_gff_file_from_staging(self, ctx, params):
```

Comment:
If you changed the function name here, you'll need to update the spec and re-run make so the associated documentation is updated